### PR TITLE
feat/interactive-ui

### PR DESCRIPTION
**Pull request by cdaringe**
# problem

ui/docs aren't interactive

# solution

make 'em interactive with meaningful examples!

demo mov

### CODE REVIEW

File: packages/effects-docs/effectsjs/home.pug [hunk 1]
```diff
@@ -44,6 +44,15 @@
 .node.funsies
   include components/logo.pug
   +logo
+.node
+  h2 Getting started
+  p There are two ways to get started:
+    ul
+      li: a(href='/effectsjs/examples.html') Try our editor and examples online, now!
+      li: a(href='/effectsjs/installation') Install effects into your javascript or typescript project
```
Comment (dpchamps) [inline packages/effects-docs/effectsjs/home.pug:52]: It may be a bold claim for typescript 😬

**Review by dpchamps — APPROVED** [review]
🔥🔥🔥

**Comment by dpchamps** [comment]
Closes #3

File: packages/effects-docs/effectsjs/home.pug (hunk 1 above)
Comment (cdaringe) [reply packages/effects-docs/effectsjs/home.pug:52]: #dreamz

**Pull request merged**